### PR TITLE
Update automated-backup-sql-2014.md

### PR DESCRIPTION
**Pull request by ujpat**

### CODE REVIEW

**Comment by Jak-MS** [comment]
@bluefooted 
- Can you review this PR?

IMPORTANT: When this content is ready to merge, you must add `#sign-off` in a comment or the approval may get overlooked.


#label:"aq-pr-triaged"
@MicrosoftDocs/public-repo-pr-review-team

**Comment by bluefooted** [comment]
@ujpat this isn't really a prerequisite or a database setting, can we move this to a note in the Settings section instead?

File: azure-sql/virtual-machines/windows/automated-backup-sql-2014.md [hunk 1]
```diff
@@ -50,6 +50,7 @@ To use Automated Backup, consider the following prerequisites:
...
 - The SQL Server VM has been registered with the SQL IaaS Agent extension in [full management mode](sql-agent-extension-manually-register-single-vm.md#upgrade-to-full). 
 -  Automated backup relies on the full [SQL Server IaaS Agent Extension](sql-server-iaas-agent-extension-automate-management.md). As such, automated backup is only supported on target databases from the default instance, or a single named instance. If there is no default instance, and multiple named instances, the SQL IaaS extension fails and automated backup will not work. 
+-  Automated backup uses [backup compression](https://docs.microsoft.com/sql/database-engine/configure-windows/view-or-configure-the-backup-compression-default-server-configuration-option?view=sql-server-ver16) by default and you cannot disable it.
```
Comment (bluefooted) [inline azure-sql/virtual-machines/windows/automated-backup-sql-2014.md:53]:
```suggestion
```

File: azure-sql/virtual-machines/windows/automated-backup-sql-2014.md [hunk 2]
```diff
@@ -50,6 +50,7 @@ To use Automated Backup, consider the following prerequisites:
...
 
 ## Settings
```
Comment (bluefooted) [inline azure-sql/virtual-machines/windows/automated-backup-sql-2014.md:56]: I would change the following paragraph to:

The following table describes the options that can be configured for Automated Backup. The actual configuration steps vary depending on whether you use the Azure portal or Azure Windows PowerShell commands. Note that Automated backup uses [backup compression](https://docs.microsoft.com/sql/database-engine/configure-windows/view-or-configure-the-backup-compression-default-server-configuration-option?view=sql-server-ver16) by default and you cannot disable it.

**Review by bluefooted — CHANGES_REQUESTED** [review]

**Comment by ujpat** [comment]
@MashaMSFT  can you help to merge this please?

**Comment by bluefooted** [comment]
#sign-off

**Review by bluefooted — APPROVED** [review]

**Comment by bluefooted** [comment]
#sign-off

**Pull request merged**